content/assignments/techjobs-console/_index.md

```diff
@@ -0,0 +1,477 @@
+1. When you think you've completed a task, run the individual test that corresponds to the task. 
+1. If the test fails, review the test output and go back to your code to try to fix it.
+1. Once the single test passes, run *all* of the tests to make sure you didn't break any tests that previously passed.
+1. Repeat this process until all tests pass. 
+
+Now we'll outline the tasks for your first apprenticeship assignment.
+
+### Implement `PrintJobs()`
+
+When trying out the program, and later when reading the code, you
+hopefully noticed that there's some work to do in the `PrintJobs()`
+method. As it stands, it currently just prints a message:
+`"PrintJobs is not implemented yet"`.
+
+Complete this method. It should print out jobs *in this precise format*:
+
+```bash
+   *****
+   position type: Data Scientist / Business Intelligence
+   name: Sr. IT Analyst (Data/BI)
+   employer: Bull Moose Industries
+   location: Saint Louis
+   core competency: Statistical Analysis
+   *****
+
+   *****
+   position type: Web - Back End
+   name: Ruby specialist
+   employer: LaunchCode
+   location: Saint Louis
+   core competency: Javascript
+   *****
+```
+
+For the autograding script to correctly grade your code, you'll need to match this format *exactly*. In particular, note the number of asterisks surrounding each listing, and the blank line between listings.
+
+If there are no results, it should print `No results`. Again, you should use this *exact* message.
+
+{{% notice orange "Warning" "rocket" %}}
+
+   To create new lines for your output, use `Environment.NewLine`.  
+
+{{% /notice %}}
+
+Using the `Environment.NewLine`  will allow the autograding unit tests pass regardless of your operating system.  `\n` is a new line in Mac OS, but will be read as `\r\n` in Windows.    Read about the differences in line breaks [here](https://dev.to/pieter/why-windows-uses-rn-newlines-instead-of-n-126l).  Read more about how `Environment.NewLine` [works](https://www.dotnetperls.com/newline).
+
+{{% notice blue "Note" "rocket" %}}
+
+   To do this, you'll need to iterate over a `List` of jobs. Each
+   job is itself a `Dictionary`. While you can get each of the items out of
+   the `Dictionary` using the known keys (`employer`, `location`, etc.),
+   think instead about creating a nested loop to loop over each
+   `Dictionary`. If a new field is added to the job records, this approach
+   will print out the new field without any updates to `PrintJobs()`.
+
+{{% /notice %}}
+
+Test this method before moving on to your next step:
+
+1. Save your changes.
+1. Run the project.
+1. Select "1" to list the jobs, and then "0" to list them all.
+1. Make sure the printout matches the styling above.
+1. Test that it prints a descriptive message if no jobs are found by selecting
+   "0" to search and then "3" to search for a location. Then enter a location
+   that is not in the data (e.g. "Cancun"). Your message should be displayed.
```

Comment:
This seems to be the test manually part of the recommended steps.  Please rework Implement Print Jobs section to flow better and align with the steps students should follow. Based on lines 338 - 342.

Reply:
Done! This is truly the last stage of the task so it should stay at the bottom.

Reply:
Is there a test they should run and pass at this point?  The lines I was referencing have moving to the Assignment #1: Tech Jobs Console lines 79 to 83.  This seems to just be step 1 what about step 2 -5? Does that not apply in this instance? 

1. Write the code for the task, verify manually that your code works by running the TechJobsConsoleAutograded project.
2. When you think you’ve completed a task, run the individual test that corresponds to the task.
3. If the test fails, review the test output and go back to your code to fix the bug.
4. Once the single test passes, run all of the tests to make sure you didn’t break any tests that previously passed.
5. Repeat this process until all tests pass.

Reply:
Gotcha! Added a note to this effect.